pkg/llmvalidate/llmvalidate.go

```diff
@@ -0,0 +1,307 @@
+package llmvalidate
+
+import (
+	"bufio"
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"unicode/utf8"
+
+	"github.com/danwakefield/fnmatch"
+	"github.com/google/generative-ai-go/genai"
+	"github.com/grafana/plugin-validator/pkg/logme"
+	"google.golang.org/api/option"
+)
+
+var ignoreList = []string{
+	".**",
+	"node_modules/**",
+	"dist/**",
+	"*.lock",
+	"**/external/**",
+	"tests/**",
+	"**/__mocks__/**",
+	"**.test.*",
+	"test-**",
+	"jest.config.*",
+	"babel.config.*",
+	"jest-setup.*",
+	"playwright.config.*",
+	"vite.config.*",
+	"tsconfig.*",
+	"package-lock.json",
```

Comment:
Is this required? `.json` is not in the `allowedExtensions` list anyways :thinking:

Reply:
Ah! left over from a previous implementation. THanks for catching it.